mautic/mautic
# Fix don't increase the verbosity of the `swiftmailer:spool:send` command run by the `mautic:emails:send` command

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



The verbosity level doesn't apply to the `swiftmailer:spool:send` command when the command is run by the `mautic:emails:send` command. This PR fixes that.
#### Steps to reproduce:
1. Set the `How should email be handled?` setting to `queue`
2. Create a segment email and send the email
3. Execute the command to process the queue with the verbosity option, e.g.  `ddev exec bin/console mautic:emails:send -vvv`
4. The message of the `swiftmailer:spool:send` command are not displayed.

![image](https://user-images.githubusercontent.com/96085911/197852659-e17ae381-f8f2-4bf8-b893-ba073018c173.png)

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Use the same step as to reproduce the issue
3. The messages of the `swiftmailer:spool:send` command are displayed .

![image](https://user-images.githubusercontent.com/96085911/197852244-885d3b93-5092-48af-ac0a-2dcd42d6e538.png)



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11629"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
